app/apollo/apollo-octopus-public/src/main/graphql/com/hedvig/android/apollo/octopus/graphql/claimflow/FragmentClaimFlowStepFragment.graphql

```diff
@@ -90,21 +90,48 @@ fragment FlowClaimSingleItemStepFragment on FlowClaimSingleItemStep {
 
 fragment FlowClaimSingleItemCheckoutStepFragment on FlowClaimSingleItemCheckoutStep {
   id
-  price {
+  singleItemStep {
+    ...FlowClaimSingleItemStepFragment
+  }
+  availableCheckoutMethods {
+    id
+    ...CheckoutMethodFragment
+  }
+  compensation {
+    ...CompensationFragment
+  }
+}
+
+fragment CompensationFragment on FlowClaimSingleItemCheckoutCompensation {
+  id
+  ...CheckoutRepairCompensationFragment
```

Comment:
Don't know if I am being too extra here 😅 But these two also are used in one place only. And I also wonder if we can defer creating them until we have a use case where we do in fact want to use this type in more than one place?